recipe-server/client/control/components/Fields.js

```diff
@@ -50,34 +50,43 @@ buildControlField.propTypes = {
   children: pt.node,
 };
 
+export const CheckboxGroup = ({ name, onChange, options = [], value }) => {
+  /**
+   * Checkbox change event handler. Appends or removes the selected checkbox's
+   * value to the existing `value` prop, and reports the change up to redux-form.
+   *
+   * @param  {Event} onChange event object
+   */
+  const handleChange = ({ event: target }) => {
+    if (target.checked) {
+      // Use Set to remove any dupes from the new value array
```

Comment:
These comments are getting pretty verbose; every line in this function has a comment. We can probably cut down on them. I think all of the lines in this function are well-written enough to not need comments, and there's no weird side effects